maildroid/tests/test_tasks.py

```diff
@@ -0,0 +1,87 @@
+import datetime
+from unittest.mock import patch
+
+from django.conf import settings
+from django.test import TestCase
+from django.utils import timezone
+
+from activitylauncher.constants import ActivityStatus
+from activitylauncher.models import Submission
+from maildroid.models import EmailLog
+from maildroid.tasks import checkout_pending_tasks
+from maildroid.constants import EmailStatus
+
+
+class TestTasks(TestCase):
+    def setUp(self):
+        settings.CELERY_TASK_ALWAYS_EAGER = True
+
+    @patch("maildroid.tasks.send_emails.delay")
+    def test_checkout_pending_task_to_send_reminders_to_start_mails(
+        self, mocked_send_emails
+    ):
+        # Creating submission objects having gap of 1-7 days between invitation creation datetime and todays date.
+        # Given gap pattern is [1,3,6] so out of these 6 objects created
+        # only 3 objects are eligible for reminders to start the activity.
+        for day in range(1, 7):
+            submission = Submission.objects.create(
+                invitation_creation_dateandtime=timezone.now()
+                - datetime.timedelta(days=day)
+            )
+            EmailLog.objects.create(
+                activity_uuid=submission.activity_uuid,
+                date_of_mail=submission.invitation_creation_dateandtime,
+                mail_status=EmailStatus.SENT.value,
+            )
+
+        # 6 invitaion mails after creating submission objects
+        self.assertEqual(mocked_send_emails.call_count, 6)
+
+        checkout_pending_tasks.apply()
+        # 3 more reminders to start which are in the gap of 1,3 and 6
+        self.assertEqual(mocked_send_emails.call_count, 9)
```

Comment:
check that the call is being made correctly as well by verifying the parameters of the call